chainer/functions/loss/sigmoid_cross_entropy.py

```diff
@@ -67,14 +79,21 @@ def sigmoid_cross_entropy(x, t, use_cudnn=True, normalize=True):
             determines the normalization constant. If true, this function
             normalizes the cross entropy loss across all instances. If else,
             it only normalizes along a batch size.
+        reduce (bool): Variable holding a boolean value which
+            determines whether to reduce the shape of the input.
+            If true, it computes the sum of cross entropy
+            and normalize it according to ``normalize`` option.
+            If else, this function computes cross entropy for each instance
+            and does not normalize it (``normalize`` option is ignored).
 
     Returns:
-        Variable: A variable object holding a scalar array of the cross entropy
-            loss.
+        Variable: A variable object holding an array of the cross entropy.
```

Comment:
Please write the behavior when `reduce` is `True` (or `'mean'`) and some values in `t` are `-1`. Specifically, the values in the output correspond to ignored labels. I think `0` or undefined is appropriate.